weaviate/connect/integrations.py

```diff
@@ -103,29 +105,28 @@ def openai(
             base_url=base_url,
         )
 
-    # not yet implemented
-    # @staticmethod
-    # def aws(
-    #     *,
-    #     access_key: str,
-    #     secret_key: str,
-    #     requests_per_minute_embeddings: Optional[int] = None,
-    #     tokens_per_minute_embeddings: Optional[int] = None,
-    # ) -> _IntegrationConfig:
-    #     return _IntegrationConfigAWS(
-    #         access_key=access_key,
-    #         secret_key=secret_key,
-    #         requests_per_minute_embeddings=requests_per_minute_embeddings,
-    #         tokens_per_minute_embeddings=tokens_per_minute_embeddings,
-    #     )
+    @staticmethod
+    def aws(
+        *,
+        access_key: Optional[str] = None,
+        secret_key: Optional[str] = None,
+        session_token: Optional[str] = None,
+        # requests_per_minute_embeddings: Optional[int] = None,
+        # tokens_per_minute_embeddings: Optional[int] = None,
```

Comment:
Are these parameters not supported by the backend yet? Will they be added back in at a later date?